### DISCUSSION TITLE
"Search In Progress" indicator

### DISCUSSION THREAD

**Discussion by GSVil** [music-assistant/support#2717] [Feature requests and ideas]
One thing I find frustrating is knowing if my music/artist/album search is actually doing anything. This is especially a problem with external music providers where it could take a few seconds to get results. It would be great to see a turning circle or hourglass that indicates that the search is in progress.

**Comment by marcelveldt** [comment]
this has been added in 2.2.0

**Reply by tonka3000** [reply]
Hey @marcelveldt ,

I'm on 2.3.4 server and I don't see any loading indicator when the query take time.
When the data is cached it is pretty fast, but if not it take sometimes multiple seconds (up to 10) to get an answer, and it looks like there is no loading at all.

I'm using the normal search menu. I type something non-cached like "Esel" which takes time. Would be nice to see some loading indicator under the "Type here to search" field to make it obvious that it is already search, maybe when taking long than a seconds or so.

Thanks in advance.

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
